src/components/ebay-table/examples/sort.marko

```diff
@@ -0,0 +1,56 @@
+import data from "./data.json";
+
+<ebay-table on-sort("emit", "sort") ...input>
+    <@header name="sellerCol" column-type="row-header" sort=("asc" as const)>
+        Seller
+    </@header>
+    <@header name="itemCol" sort>
```

Comment:
Why do we have a simple sort for each column? Seems somewhat unnecessary.
I think for each column that is sorted I can pass in sort as needed from the `on-sort` method (we should demonstrate this). I assume if sort is not passed then its not going to be sorted.

Reply:
If `sort` is `undefined | false`, no sorting function for the column. I think we discussed this before, that whether we want `sortable` + `sortOrder` or just simple `sort` and we decided to use simple `sort`. Main functionality here is to distinguish columns that can be sorted or not.

Reply:
I think in this example we should change this to pass in the sort value from the `on-sort`. This example here isn't a good example and can be misleading.

Reply:
Updated example please check again thanks